Nodejs-Bug-Bounty-Security-Fund.md

```diff
@@ -0,0 +1,34 @@
+# Node.js Bug Bounty/Security Fund
+
+The TSC maintains a LFX Crowdfunding account in order to accept
```

Comment:
Will people know what LFX is? (I don't, unless it's Linux Foundation, in which case maybe we should spell that out?)

Reply:
expanded